server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -598,15 +598,23 @@ export class ScribeLambda implements IPartitionLambda {
 			this.pendingCheckpointOffset = queuedMessage;
 			return;
 		}
-
+		let databaseCheckpointFailed = false;
 		this.pendingP = clearCache
 			? this.checkpointManager.delete(this.protocolHead, true)
-			: this.writeCheckpoint(checkpoint);
+			: this.writeCheckpoint(checkpoint).catch((error) => {
```

Comment:
My understanding is that this is now:
```ts
this.writeCheckpoint(checkpoint).catch().then().catch()
```
So that would mean that we are eating the first `catch` error silently. Should we log that?

Reply:
Added a log for the error in the first `catch`